pygeoapi/provider/postgresql.py

```diff
@@ -298,6 +336,8 @@ def get(self, identifier):
             row_data = cursor.fetchall()[0]
             feature = self.__response_feature(row_data)
 
+            feature['previous'] = self.get_previous(cursor, identifier)
```

Comment:
Suggest changing to `feature['prev']`.

Reply:
We could and in the interest of brevity I agree. However, in the interest of readability it might be better to stay 'previous'. I don't really mind though. Feel free to change it if you prefer or let me know if you want me to do it.

Reply:
If you can change to `prev` that would be consistent with OGC API - Features proper. Thanks.

Reply:
Thanks @tomkralidis! Your points absolutely make sense. It shall be done. :-)